getting-started/vs-code-integration/dpl-wizard.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Configure Document Processing Libraries 
+page_title: Document Processing Libraries Integration
+description: Learn how to add a reference to the Telerik Document Processing Libraries(DPL) packages in an existing Telerik Ui for Blazor application using the Document Processing Libraries Wizard.
+slug: getting-started-vs-code-integration-dpl
+position: 5
+---
+
+# Configuring Document Processing Libraries
+
+This article demonstrates how to add a reference to the [Telerik Document Processing Libraries (DPL)](https://www.telerik.com/document-processing-libraries) packages in an existing Telerik UI for Blazor application using the Document Processing Libraries Wizard. The wizard is included in the [Telerik UI for Blazor Visual Studio Code extension](https://marketplace.visualstudio.com/items?itemName=TelerikInc.blazortemplatewizard) and enables you to easily add the required packages when working with the Telerik Document Processing Library.
```

Comment:
I recommend using the complete and correct name of the extension at least in its first instance on the page.

It's `Telerik UI for Blazor Productivity Tools`.

Reply:
Ideally, go with `<a href="..." target="_blank">` for your external links. See [here](https://docs.telerik.com/style-guide/cross-references#guidelines).